Makefile

```diff
@@ -62,24 +62,28 @@ $(BUILDDIR)/$(BINARY_NAME): $(GOFILES) | $(BUILDDIR)
 # Tools
 
 GOLINT = $(GOBIN)/golint
-$(GOBIN)/golint: | $(BASE) ; $(info  Installing golint...)
-	$Q go install golang.org/x/lint/golint
+$(GOLINT): | $(BASE) ; $(info  Building golint...)
```

Comment:
Should we look to update to golangci-lint?
golint is deprecated.

Reply:
You are right! And `golang-ci` is already in place in job lanes ([static-scan.yml](https://github.com/k8snetworkplumbingwg/sriov-cni/blob/d5784b16ca7c5fdb069430a1ff25510c37cafcfc/.github/workflows/static-scan.yml#L10)).

So it's better to remove `lint` rule from `test-coverage` dependencies list, as it may have a different output from golang-ci.

BTW, removing golint entirely from the project should be done in a different PR, do you agree?

Reply:
Yes, makes sense to me.
Would you be able to create this PR? 
This we can look to merge that PR, aswell as this.